toolchain/lex/tokenized_buffer_test.cpp

```diff
@@ -17,14 +17,12 @@
 #include "toolchain/diagnostics/mocks.h"
 #include "toolchain/lex/lex.h"
 #include "toolchain/lex/tokenized_buffer_test_helpers.h"
+#include "toolchain/testing/compile_helper.h"
 #include "toolchain/testing/yaml_test_helpers.h"
 
-namespace Carbon::Lex {
+namespace Carbon::Testing {
```

Comment:
For some reason, when we switched to using the `Lex` sub-namespace, I thought we had ended up switching how tests work instead of putting them in the `Testing` namespace. But I can't find such a discussion, only the discussion suggesting putting the tests in `Testing` as you've switched to here.

Anyways, I'm happy either way, only mentioning it in case there *was* such a discussion, and I just can't find it but you or someone else remember it.

Reply:
I've tracked down discussion [here](https://discord.com/channels/655572317891461132/655578254970716160/1141153794097422457), via https://github.com/carbon-language/carbon-lang/pull/3170/files#r1310772837. I'm updating #3070 to mention this.

Reply:
But now I see #3244, I guess there was a little more.

Reply:
Undid this part of the change